### ISSUE TITLE
add kraken2?

### ISSUE THREAD

**Issue by kapsakcj** [StaPH-B/docker-builds#65]
I started a dev branch for this, but didn't get very far....don't know if anyone is interested in this or not. 

One minor issue is that there are no releases for kraken2, only pre-releases: https://github.com/DerrickWood/kraken2/releases

**Comment by kapsakcj** [comment]
After a second glance, it looks like I did get pretty far with the Dockerfile. https://github.com/StaPH-B/docker-builds/blob/curtis-dev/kraken2/2.0.7-beta/Dockerfile

One thing to change, would be to move the `RUN ..` lines that downloads the db to the very end of the dockerfile.

I have not tested this image, but would welcome anyone that wants to try!

**Comment by kapsakcj** [comment]
PR started #66 . Although I can't test the build for this image since the FTP site for downloading the minikraken2 db is blocked on my network. Fingers-crossed that it builds ok!

**Comment by kapsakcj** [comment]
PR #66 merged into master. Building on dockerhub now. Haven't tested with real data yet...

**Comment by kapsakcj** [comment]
Will add to README.md and Program_licenses.md once it is tested. If it tests OK, then I'll add another dockerhub tag/build `staphb/kraken2:2.0.8-beta`

**Comment by kapsakcj** [comment]
kraken2 via docker worked great and produced the expected output. closing this issue.

Has been hadded to readme.md and program_licenses.md
